benchmarks/shared/src/commonMain/kotlin/io/realm/kotlin/benchmarks/Entities.kt

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2022 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.kotlin.benchmarks
+
+import io.realm.RealmInstant
+import io.realm.RealmList
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import io.realm.realmListOf
+
+enum class SchemaSize {
+    SINGLE, SMALL, LARGE
+}
+
+val SINGLE_SCHEMA = setOf(Entity1::class)
```

Comment:
I would put these in a map by the above keys, so that we can easily lookup the correct schema from the `SchemaSize` programatically.

Reply:
Good idea 👍